SECURITY.md

```diff
@@ -1,10 +1,25 @@
 # Security Policy
 
+## Overview
+
+This policy applies to MDN's website (`developer.mozilla.org`), backend services, and GitHub repositories in the [`mdn`](https://github.com/mdn) organization. Issues affecting other Mozilla products or services should be reported through the [Mozilla Security Bug Bounty Program](https://www.mozilla.org/en-US/security/bug-bounty/).
+
+For non-security issues, please file a [content bug](https://github.com/mdn/content/issues/new/choose), a [frontend bug](https://github.com/mdn/fred/issues/new/choose) or a [content/feature suggestion](https://github.com/mdn/mdn/issues/new/choose).
```

Comment:
Should this be more generic than "frontend bug"? As currently written, a user with a backend bug has nowhere to file it. Perhaps something like: (if we consider fred the triage repo for all engineering issues if the user doesn't know where to file it)
```suggestion
For non-security issues, please file a [content bug](https://github.com/mdn/content/issues/new/choose), an [engineering bug](https://github.com/mdn/fred/issues/new/choose) or a [content/feature suggestion](https://github.com/mdn/mdn/issues/new/choose).
```

Reply:
Yes, I like "website bug"!